hwilib/ui/bitbox02pairing.ui

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>BitBox02PairingDialog</class>
+ <widget class="QDialog" name="BitBox02PairingDialog">
+  <property name="windowModality">
+   <enum>Qt::WindowModal</enum>
+  </property>
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>209</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Dialog</string>
```

Comment:
"BitBox02 pairing"?

Reply:
All other gui dialogs are like this, and the title is set in gui.py for all dialogs. I do it the same way for consistency.